logic/round.js

```diff
@@ -200,16 +209,26 @@ Round.prototype.applyRound = function () {
 		var changes = roundChanges.at(i);
 
 		this.scope.library.logger.trace('Delegate changes', { delegate: delegate, changes: changes });
+		var self = this;
```

Comment:
Why no do that once in constructor?

Reply:
I don't think assigning `this` to module scope variable is a good practice. If we create two instances of round, that variable will be overridden. 

Although I have updated the usage of variables in this method and extracted this variable upper level.